.github/workflows/buildtags.yml

```diff
@@ -32,9 +46,9 @@ jobs:
           images: |
             unleashorg/unleash-community-${{ matrix.folder }}
           tags: |
-            type=semver,pattern={{version}}
-            type=semver,pattern={{major}}.{{minor}}
-            type=semver,pattern={{major}}
+            type=semver,pattern={{version}},value=${{ steps.retrieve-version.outputs.version }}
+            type=semver,pattern={{major}}.{{minor}},value=${{ steps.retrieve-version.outputs.major }}.${{ steps.retrieve-version.outputs.minor }}
+            type=semver,pattern={{major}},value=${{ steps.retrieve-version.outputs.major }}
```

Comment:
Is it intentional to have to patterns with `major` and none with `patch`? It doesn't seem like you use the `patch` variable for anything?